src-worker/planing/executor.ts

```diff
@@ -58,17 +59,12 @@ export class Executor {
 
     if (target === null) {
       // Wait for targets
-      await new Promise(resolve => setTimeout(resolve, 5000))
+      await sleep(5000)
       return
     }
 
     const algorithm = this.algorithmGroup.getByType(target.method)
 
-    if (algorithm === null) {
-      await new Promise(resolve => setTimeout(resolve, 50))
```

Comment:
algorithm may be null for old version of the program

Reply:
why would we wait for 50 ms in such a case? If the `algorithm` is `null` it won't ever change to something else